### PR TITLE
[DOCS-7707] Update ACS compatibility for Alfresco Intelligence Services 3.1.3

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by HeinRagas — DISMISSED** [review]
ok

File: intelligence-services/latest/support/index.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-The following are the supported platforms for Alfresco Intelligence Services 3.1:
+The following are the supported platforms for Alfresco Intelligence Services 3.1.x:
 
 | Version | Notes |
 | ------- | ----- |
 | Content Services 23.x | |
-| Content Services 7.4 | |
+| Content Services 7.4.2 | |
+| Content Services 7.3.2 | |
+| Content Services 7.2.2 | Use the AIS 3.1.3 T-Engine with the older AIS 1.4.2 AMP (see the Release Notes for more details)|
```
Comment (mbujak-hyland) [inline intelligence-services/latest/support/index.md:12]: Do we put full stop at the end?

Comment (anxumalo) [reply intelligence-services/latest/support/index.md:12]: Yes, I should :-)

**Review by mbujak-hyland — APPROVED** [review]
LGTM

**Pull request merged**